src/popover.coffee

```diff
@@ -45,6 +45,12 @@ Ember.Widgets.BodyEventListener,
 
   didInsertElement: ->
     @_super()
+
+    # Make sure that after the popover is rendered, set focus to the first
+    # tabbable element
+    Ember.run.schedule 'afterRender', this, ->
```

Comment:
Looks like this is duplicated here and in modal. Could we move it into `TabbableModal`? Not sure in what order the `didInsertElement` hooks get run.